sos/plugins/ipa.py

```diff
@@ -56,34 +57,33 @@ def setup(self):
             "/etc/dirsrv/slapd-*/dse.ldif",
             "/etc/dirsrv/slapd-*/schema/99user.ldif",
             "/etc/hosts",
-            "/etc/named.*"
+            "/etc/named.*",
+            "/etc/pki-ca/CS.cfg",
+            "/etc/ipa/ca.crt",
+            "/etc/ipa/default.conf",
+            "/var/lib/certmonger/requests/[0-9]*",
+            "/var/lib/certmonger/cas/[0-9]*"
         ])
 
         self.add_forbidden_path("/etc/pki/nssdb/key*")
         self.add_forbidden_path("/etc/pki-ca/flatfile.txt")
         self.add_forbidden_path("/etc/pki-ca/password.conf")
         self.add_forbidden_path("/var/lib/pki-ca/alias/key*")
-
         self.add_forbidden_path("/etc/dirsrv/slapd-*/key*")
         self.add_forbidden_path("/etc/dirsrv/slapd-*/pin.txt")
         self.add_forbidden_path("/etc/dirsrv/slapd-*/pwdfile.txt")
-
         self.add_forbidden_path("/etc/named.keytab")
 
         self.add_cmd_output([
             "ls -la /etc/dirsrv/slapd-*/schema/",
-            "ipa-getcert list",
```

Comment:
If the user doesn't have a valid kerberos ticket for the replication user It tries to do interactive auth (user/pass) at the terminal and hits the command timeout.